### ISSUE TITLE
Stop abusing sudo

### ISSUE THREAD

**Issue by hongquan** [fossasia/susi_linux#240]
**Describe the bug**
In https://github.com/fossasia/susi_linux/blob/master/access_point/server/server.py, I see a lot of use of `sudo`. Why? 

Normal application should not be run with "root" permission.

Is it because it need to write to GPIO? If so, just make the current user `pi` have the write permission on that specific GPIO port.

**Comment by norbusan** [comment]
Indeed, not only server.py, but also lots of install routines and other parts are filled with sudos, which is not optimal. I will slowly look through it and see if I can get rid of some.
